rdrf/rdrf/management/commands/set_registry_version.py

```diff
@@ -0,0 +1,15 @@
+from django.core.management import BaseCommand
+from rdrf.models.definition.models import Registry
+
+
+class Command(BaseCommand):
+    help = "Set the registry version"
+
+    def handle(self, *args, **options):
```

Comment:
Make more specific as we do have the possibility of having more that one registry on a site.

Reply:
modified as suggested